Pahappa-LTD/call-center-ticketing-system
# Ft get ticket by status

What does this PR do?
This filters out tickets by status and only displays on the screen the required tickets filtered out by status.
Description of the task to be completed
This feature provides a convenient way to filter and view tickets that are categorized under specific status types, that is "Open," "In Progress," or "Resolved."
How should this be manually tested
-Ensure that there are tickets available in the system with various status types (Open, In Progress, Resolved) to test the filtering functionality effectively.
-Choose a specific status from the available options
-Try placing a number beyond the given options and an Invalid message will be displayed.
-Verify that the system retrieves and displays the tickets matching the selected status. The displayed tickets should be filtered according to the chosen status criteria.
-Repeat the testing process with different status options to ensure that the filtering mechanism works correctly for all status types.
Screenshots
![Screenshot 2023-07-07 174041](https://github.com/Pahappa-LTD/call-center-ticketing-system/assets/98257312/89d8b4ca-6513-4c95-9789-d8df438d273d)
![Screenshot 2023-07-07 174123](https://github.com/Pahappa-LTD/call-center-ticketing-system/assets/98257312/7289e84a-3b8c-40fa-9cc7-f7e2b5767360)
